### ISSUE TITLE
Get the latest or trending posts from username/publication isn't working

### ISSUE THREAD

**Issue by hcarneiro** [frandiox/medium-json-feed#3]
When using `@user-name/latest`, `publication-name/latest` or `publication-name/trending` we get a `403` error.

You can check by testing [your own example](https://medium-json-feed.herokuapp.com/@mikeal/latest).

I believe that's because Medium has a weird check before showing the content.

**Comment by frandiox** [comment]
I guess they changed the permissions of their API. I'm 100% sure that example was working before. It still works with `publication-name/trending`, though.

I think there's nothing we can fix here, other than removing the example :/

**Comment by hcarneiro** [comment]
@frandiox Yeah it was working, I had it working on my site, that's how I noticed.

Well, no worries, feel free to close the ticket if there's nothing we can do, just thought this was the best way to let you know.

**Comment by alvimuh** [comment]
@hcarneiro just remove "/latest" from  the url. It's work for me. But not all post are retrieved
